20-api-v2/acl.md

```diff
@@ -9,11 +9,11 @@ BearerToken allows to attach signed Extended ACL rules to the request in
 `RequestMetaHeader`. If container's Basic ACL rules allow, the attached rule
```

Comment:
`20-api-v2` is also autogenerated from `neofs-api` repo. Please move corrections there.